### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by quentinlesceller**
~~Was reading through core code and *had* to fix these typos.~~
Thought I would check the entire code for typos.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ Please be kind and courteous. There’s no need to be mean or rude. Respect that
 
 Please keep unstructured critique to a minimum. If you have solid ideas you want to experiment with, make a fork and see how it works.
 
-We will exclude you from interaction if you insult, demean or harass anyone. That is not welcome behaviour. We interpret the term “harassment” as including the definition in the [Citizen Code of Conduct](http://citizencodeofconduct.org/); if you have any lack of clarity about what might be included in that concept, please read their definition. In particular, we don’t tolerate behavior that excludes people in socially marginalized groups.
+We will exclude you from interaction if you insult, demean or harass anyone. That is not welcome behavior. We interpret the term “harassment” as including the definition in the [Citizen Code of Conduct](http://citizencodeofconduct.org/); if you have any lack of clarity about what might be included in that concept, please read their definition. In particular, we don’t tolerate behavior that excludes people in socially marginalized groups.
```
Comment (antiochp) [inline CODE_OF_CONDUCT.md]: Depends if we are using US spelling 😄

File: core/src/core/pmmr.rs [hunk 2]
```diff
@@ -269,7 +269,7 @@ where
 	/// The last position in the PMMR
 	pub last_pos: u64,
 	backend: &'a mut B,
-	// only needed for parameterizing Backend
+	// only needed to parametrise Backend
```
Comment (antiochp) [inline core/src/core/pmmr.rs]: I'm pretty sure "parametrise" is not right?
Old and not in common use any more?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (yeastplume) [reply CODE_OF_CONDUCT.md]: I'm okay with standardizing on American spelling throughout, it's generally the path of least resistance.

File: core/src/core/pmmr.rs (hunk 2 above)
Comment (yeastplume) [reply core/src/core/pmmr.rs]: Not right, just a missing 'e' typo

Comment (quentinlesceller) [reply core/src/core/pmmr.rs]: Both are correct but I will update for ```parameterize```.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (quentinlesceller) [reply CODE_OF_CONDUCT.md]: No strong opinion on that one. Let me know what you prefer.

Comment (ignopeverell) [reply CODE_OF_CONDUCT.md]: Even if I try hard, I'm pretty sure I'll let some US spelling slip through... for the words I can spell, that is.

**Pull request merged**